aiida_quantumespresso/calculations/__init__.py

```diff
@@ -203,8 +267,12 @@ def prepare_for_submission(self, folder):
         calcinfo = datastructures.CalcInfo()
 
         calcinfo.uuid = str(self.uuid)
-        # Empty command line by default
-        cmdline_params = settings.pop('CMDLINE', [])
+        # Start from an empty command line by default
+        cmdline_params = self._add_parallelization_flags_to_cmdline_params(
+                cmdline_params=settings.pop('CMDLINE', [])
```

Comment:
overindented no?
```suggestion
            cmdline_params=settings.pop('CMDLINE', [])
```

Reply:
Yeah.. we should fix the issue that makes the formatting not work on this file.